### ISSUE TITLE
response GET api/v1/discussions/<id> is invalid

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#278]
swagger shows that I should receive this kind of response:

```
{
  "attachment_count": 0,
  "date_insert": "2017-04-28T12:43:04.374Z",
  "date_update": "2017-04-28T12:43:04.374Z",
  "discussion_id": "string",
  ...
}
```

but actually it is wrapped in a discussion property:

```
{
	"discussion": {
		"total_count": 0,
		"excerpt": "Your email client cannot read this email.\nTo view it online, please go here:\nhttp://162.244.67.108/~client/display.php?M=11664742&C=94c9fa492a809f4120021467a9b09dd2&S=10942&L=165&N=3641\n\n\nTo stop rece",
		"date_insert": "2017-04-28T19:24:25.580000+00:00",
		"unread_count": 0,
		"discussion_id": "b0e7431b-602c-4a23-9fed-68536eaa9e16",
		"attachment_count": 0,
		"importance_level": 92
	}
}
```

**Comment by sapiens-sapide** [comment]
OK, I'll fix.

**Comment by sapiens-sapide** [comment]
Closed by commit #e55f983.
